Sdk/Conversation/SwrveConversationStarRatingView.m

```diff
@@ -0,0 +1,200 @@
+#import "SwrveConversationStarRatingView.h"
+#import "SwrveSetup.h"
+
+@interface SwrveConversationStarRatingView ()
+
+@property (strong, nonatomic) UIImage *swrveNotSelectedImage;
+@property (strong, nonatomic) UIImage *swrveFullSelectedImage;
+@property (strong, nonatomic) UIColor *swrveBackgroundColor;
+@property (strong, nonatomic) UIColor *swrveStarColor;
+@property (readwrite, nonatomic) float swrveCurrentRating;
+@property (readwrite, nonatomic) NSUInteger swrveMaxRating;
+@property (readwrite, nonatomic) float swrveLeftMargin;
+@property (readwrite, nonatomic) float swrveMidMargin;
+@property (strong, nonatomic) NSMutableArray * swrveStarViews;
+@property (assign, atomic) CGSize swrveMinImageSize;
+
+@end
+
+@implementation SwrveConversationStarRatingView
+
+@synthesize swrveStarViews;
+@synthesize swrveNotSelectedImage;
+@synthesize swrveFullSelectedImage;
+@synthesize swrveStarColor;
+@synthesize swrveBackgroundColor;
+@synthesize swrveCurrentRating;
+@synthesize swrveMaxRating;
+@synthesize swrveMidMargin;
+@synthesize swrveLeftMargin;
+@synthesize swrveMinImageSize;
+@synthesize swrveRatingDelegate;
+
+
+- (id) initWithDefaults {
+    self =[super init];
+    if(self){
+        self.swrveStarViews = [[NSMutableArray alloc] init];
+        self.swrveNotSelectedImage = [UIImage imageNamed:@"star-empty.png"];
+        self.swrveFullSelectedImage = [UIImage imageNamed:@"star-full.png"];
+        self.swrveStarColor = [UIColor redColor];
+        self.swrveCurrentRating = 0;
+        self.swrveMaxRating = 5;
+        self.swrveMidMargin = 5.0f;
+        self.swrveLeftMargin = 0.0f;
+        self.swrveMinImageSize = CGSizeMake(15, 15);
+        
+        [self setStarsFromMax];
+    }
+    return self;
+}
+
+- (void) setDelegate:(id<SwrveConversationStarRatingViewDelegate>)delegate {
+    self.swrveRatingDelegate = delegate;
+}
+
+- (void) setStarsFromMax {
+    [self.swrveStarViews removeAllObjects];
+    // Add new image views
+    for(NSUInteger i = 0; i < self.swrveMaxRating; ++i) {
+        UIImageView *imageView = [[UIImageView alloc] init];
+        imageView.contentMode = UIViewContentModeScaleAspectFit;
+        [self.swrveStarViews addObject:imageView];
+        [self addSubview:imageView];
+    }
+    [self setNeedsLayout];
+    [self refresh];
+}
+
+- (void) initRatingTypewithStarColor:(UIColor *) starColor withBackgroundColor:(UIColor *)backgroundColor
```

Comment:
Can we change the name to other than init*? It does not call [super init] (and it shouldn't)